subprojects/docs/src/docs/release/notes.md

```diff
@@ -181,6 +181,37 @@ This allows configuration of PMD to run its analysis on more than one thread.
 
 See the [documentation](userguide/pmd_plugin.html#sec:pmd_conf_threads) for more information.
 
+### Configuration cache improvements
+
+#### Modification of system properties at the configuration time
```

Comment:
This is about a bug fixed. I wonder if we should mention it in the release notes, probably not.

Reply:
It's borderline, IMO, but I don't have a strong enough opinion about this one.